src/entities/SurveyTopic/model.ts

```diff
@@ -0,0 +1,28 @@
+import { Model } from 'decentraland-gatsby/dist/entities/Database/model'
+import { SQL, conditional, table } from 'decentraland-gatsby/dist/entities/Database/utils'
+
+import { ProposalAttributes, ProposalType } from '../Proposal/types'
+import { getProposalCategory } from '../Proposal/utils'
+import ProposalSurveyTopicModel from '../ProposalSurveyTopics/model'
+
+import { SurveyTopicAttributes, Topic } from './types'
+
+export default class SurveyTopicModel extends Model<SurveyTopicAttributes> {
+  static tableName = 'survey_topics'
+  static withTimestamps = false
+  static primaryKey = 'id'
+
+  static async getSurveyTopic(
+    proposalType: ProposalType,
+    proposalConfiguration: Pick<ProposalAttributes, 'configuration'>
+  ): Promise<Topic[]> {
+    const proposalCategory = getProposalCategory(proposalType, proposalConfiguration)
+    return await this.query(SQL`
+    SELECT s.topic_id
+    FROM ${table(SurveyTopicModel)} s
+        INNER JOIN ${table(ProposalSurveyTopicModel)} ps ON ps."topic_id" = s."topic_id"
+        WHERE ps."proposal_type" = ${proposalType}
```

Comment:
Are we validating this is a valid proposal type?

Reply:
Good question!

This method is only invoked with the `proposal.type` we have in our DB, so there shouldn't be a problem

```typescript
     const proposal = await ProposalModel.getProposal(id)
     return await SurveyTopicModel.getSurveyTopic(proposal.type, proposal.configuration)
```